src/main/java/rotateit/github/domain/Label.java

```diff
@@ -0,0 +1,16 @@
+package rotateit.github.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class Label {
+    private final String name;
+
+    @JsonCreator
+    public Label(String name) {
```

Comment:
ooook
found this
```
NOTE: when annotating creator methods (constructors, factory methods), method must either be:

Single-argument constructor/factory method without JsonProperty annotation for the argument: if so, this is so-called "delegate creator", in which case Jackson first binds JSON into type of the argument, and then calls creator
```
here http://static.javadoc.io/com.fasterxml.jackson.core/jackson-annotations/2.8.9/com/fasterxml/jackson/annotation/JsonCreator.html
so I guess we're good ;)